README.md

```diff
@@ -72,15 +72,16 @@ It also has several useful helpers to handle errors.
 
 | Param | Type | Default | Description |
 | --- | --- | --- | --- |
-| auth | `Object <username, password, targetBranch>` | - | See `auth` description [below](#auth). |
+| auth | `Object <username, password, targetBranch, pcc, overridePcc>` | - | See `auth` description [below](#auth). |
```

Comment:
@dchertousov it is the effective pcc of credentials being used. It is copied into each of the search results and not used anywhere else; it is a sort of previous design artefact.
We can remove it, then users should control which pcc they're using by external logic, which fits in perfectly with overridePcc.